src/yang/ietf-bgp.yang

```diff
@@ -851,9 +851,12 @@ module ietf-bgp {
               "RFC 4271, Section 8.2.2.";
           }
           description
-            "The backward-transition event is
+            "The backward-transition event is generally
```

Comment:
Don't include this in the fix build commit.

Reply:
I believe this came to me as part of merge I did with v13.

Reply:
On re-checking, apparently not. Will remove it.